docs/component.md

```diff
@@ -90,59 +90,62 @@ element by calling `new` on the component’s constructor function:
 
 ```js
 const HelloWorld = Component.extend({
-  tag: "hello-world",
-  view: "Hello <content>world</content> <ul>{{#each(items)}} {{>itemPartial}} {{/each}}</ul>",
-  ViewModel: {
-    items: {
-      default: () => []
-    }
-  }
+	tag: "hello-world",
+	view: `
+		<can-slot name="greetingTemplate" />
+		<content>world</content>
+		<ul>{{#each(items)}} {{this}} {{/each}}</ul>
```

Comment:
I think it would be nice to use the new spacing rules: `{{# each(items) }} {{ this }} {{/ each}}`.  But not important.

Reply:
Rules? Are they documented anywhere? This is where I looked: https://canjs.com/doc/guides/contributing/code.html#Styleguide